BBHSE/new.php

```diff
@@ -0,0 +1,263 @@
+<?php
+//by Art Eaton
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+formHeader("Form: BBHSE");
+?>
+<html><head>
+<?php html_header_show();?>
+<link rel=stylesheet href="<?echo $css_header;?>" type="text/css">
+</head>
+<body <?echo $top_bg_line;?> topmargin=0 rightmargin=0 leftmargin=2 bottommargin=0 marginwidth=2 marginheight=0>
+<form method=post action="<?echo $rootdir;?>/forms/BBHSE/save.php?mode=new" name="my_form">
+<br>
+<span class="title"><center>Brief Behavioral Health Status Examination H2010 HO</center></span><br><br>
+<center><a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[Save]</a>
+You must EDIT the form after saving and reviewing to finalize
+<a href="<?php echo $GLOBALS['form_exit_url']; ?>" class="link_submit"
+ onclick="top.restoreSession()">[Don't Save]</a></center>
+<br>
+<?php $res = sqlStatement("SELECT date,counselor_id,facility FROM form_encounter WHERE encounter = $encounter");
+$encounterdata_array = SqlFetchArray($res); ?>
+<?php $res = sqlStatement("SELECT fname,mname,lname,ss,street,city,state,postal_code,phone_home,DOB FROM patient_data WHERE pid = $pid");
+$result = SqlFetchArray($res); ?>
```

Comment:
Right.  This is not good.  I think this form is about 8 or nine years old at this point.